go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	github.com/armon/go-metrics v0.0.0-20180917152333-f0300d1749da // indirect
 	github.com/boltdb/bolt v1.3.1
 	github.com/cheggaaa/pb v1.0.5-0.20160713104425-73ae1d68fe0b
+	github.com/ckaznocha/protoc-gen-lint v0.2.1 // indirect
```

Comment:
not sure if this has to be changed. Please see if you can compile and build without any changes to go.mod and go.sum. It is an observation